js/node/hapi/main.js

```diff
@@ -0,0 +1,24 @@
+const Hapi = require('@hapi/hapi');
```

Comment:
Is this an example? Why is this in the linting project? I could understand examples that highlight why certain linting choices were made over others, but I'm not sure I see the purpose of why this file is here.

Reply:
This is not needed in this project.